modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,226 @@
 [id="creating-workspaces"]
 = Creating workspaces
 
-Rough technical info to be dumped here by the assigned SME.
+If your use case does not permit use of the {prod-short} dashboard, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster.
+
+[NOTE]
```

Comment:
@l0rd, I've moved all remaining points from the intro paragraph into the bullet list, WDYT?
![image](https://user-images.githubusercontent.com/78564304/214516261-6db6c03b-7861-4575-b0e7-e0f085d5020e.png)

Reply:
👍